auth-agents-common/src/main/java/org/apache/atlas/plugin/util/KeycloakUserStore.java

```diff
@@ -177,7 +177,7 @@ public RangerRoles loadRolesIfUpdated(long lastUpdatedTime) throws AtlasBaseExce
         List<HeraclesRoleViewRepresentation> roleRetrievalResult;
 
         do {
-            roleRetrievalResult = getHeraclesClient().getRolesMappings(roleFrom, roleSize);
+            roleRetrievalResult = getHeraclesClient().getRolesMappings(roleFrom, roleSize, new String[]{"composite_roles", "groups"});
```

Comment:
nit: in general good practice is to include enums.

Reply:
Added enums